service/service.go

```diff
@@ -96,11 +96,15 @@ func NewSociSnapshotterService(ctx context.Context, root string, config *Config,
 	if userxattr {
 		opq = layer.OverlayOpaqueUser
 	}
+	// The filesytems GetSources function will default to FromDefaultLabels for now,
+	// since there are no size labels for layers within the snapshotters package in
+	// containerd. In order to support kubernetes/CRI we will have to either add
+	// upstream support for layer size labels or make network calls within the
+	// snapshotter to fetch layer size.
 	// Configure filesystem and snapshotter
-	fsOpts := append(sOpts.fsOpts, socifs.WithGetSources(sources(
-		sourceFromCRILabels(hosts),      // provides source info based on CRI labels
+	fsOpts := append(sOpts.fsOpts, socifs.WithGetSources(
```

Comment:
Cannot put the comment in func definition

I think we can remove this func (`source`) for now since it's unused and a trivial function, and I feel it's right location should be in `fs/source/source.go`. If we need it later, we can add it there.